models.py

```diff
@@ -248,6 +247,7 @@ def evaluate(self, Data, output="Average"):
             self.model.train(True) # need this to be true to activate dropout
             with torch.no_grad():
                 outputs = torch.hstack([self.model(Data) for _ in range(self.config.proxy.dropout_samples)]).cpu().detach().numpy()
+            outputs = torch.from_numpy(outputs)
```

Comment:
why?

Reply:
Othervwise, error from mean function
TypeError: mean() received an invalid combination of arguments - got (numpy.ndarray, dim=int), but expected one of:
 * (Tensor input, *, torch.dtype dtype)

Reply:
I have seen this too which is weird. I don't know when this was added. In any case the way to fix it should be to compute np.mean() and np.std() instead of converting back to tensors.

Reply:
I will fix it myself.